mesonbuild/interpreter/primitives/integer.py

```diff
@@ -65,10 +65,13 @@ def is_even_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
     def is_odd_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
         return self.held_object % 2 != 0
 
-    @noKwargs
+    @typed_kwargs(
+        'to_string_method',
```

Comment:
Nitpicking: you can remove _method suffix in that string, the public name is to_string.